openpathsampling/ensemble.py

```diff
@@ -2212,7 +2226,8 @@ def __call__(self, trajectory, trusted=None, candidate=False):
         else:
             logger.debug("Untrusted VolumeEnsemble " + repr(self))
             # logger.debug("Trajectory " + repr(trajectory))
-            for frame in trajectory.as_proxies():
+            # This can sometimes get a list instead of a Trajectory
+            for frame in list(trajectory):
```

Comment:
I think you'll want to do something here to allow this to load as proxies instead. The current implementation will require loading the full snapshot object every time (with all its data) -- could be a significant slowdown in custom analysis. Maybe add something like:

```python
iter_traj = getattr(trajectory, 'iter_proxies', trajectory.__iter__)
for frame in iter_traj():
    ...
```